astro/src/content/json/themes/templates.json

```diff
@@ -532,7 +537,29 @@
     "displayName": "OAuth complete registration",
     "fieldName": "oauth2CompleteRegistration",
     "description": "This page contains a form that is used for users that have accounts but might be missing required fields.",
-    "path": "/oauth2/complete-registration"
+    "path": "/oauth2/complete-registration",
+    "variables": [
+      {
```

Comment:
Don't know why no fields were in here, but added the existing + new (passwordSet)